app/experimenter/nimbus-ui/src/components/PageResults/TableVisualizationRow/index.tsx

```diff
@@ -190,7 +191,8 @@ const formattedAnalysisPointComparator = (
 
 const TableVisualizationRow: React.FC<{
   metricKey: string;
-  results: BranchDescription;
+  results: BranchDescription | BranchDescriptionGrouped;
```

Comment:
When are the results not grouped?

Reply:
Yep, I will fix that. Once I moved to making the weekly graphs accept a group argument this should no longer exist.